tests/unit/payloadValidation.unitTest.ts

```diff
@@ -86,15 +81,39 @@ describe("payloadValidation", () => {
       it("should pass the validation and return the validated payload for HGV", () => {
         const techRec: any = cloneDeep(mockData[43]);
         delete techRec.techRecord[0].statusCode;
-        delete techRec.techRecord[0].createdByName;
-        delete techRec.techRecord[0].createdAt;
-        delete techRec.techRecord[0].createdById;
         const validatedPayload = validatePayload(techRec.techRecord[0]);
         expect(validatedPayload.value).toBeDefined();
         expect(validatedPayload.error).toEqual(undefined);
         expect(validatedPayload.value.vehicleType).toEqual(VEHICLE_TYPE.HGV);
       });
 
+      it("should pass the validation and return the validated payload for LGV", () => {
+        const techRec: any = cloneDeep(mockData[124]);
```

Comment:
this is an old issue, we're having bad and outdated test data in the seed file and it's complaining because it doesn't match the interface. We would need to clean the entire seed file at some point

Reply:
Could possibly start to add one interface each for the simple vehicle types such as Car, LGV and motorcycle and then refactor where necessary for the regular types (psv, trl, hgv) - logged a ticket: https://jira.dvsacloud.uk/browse/CVSB-15528